### PR TITLE
apps: upgrade prometheus-elasticsearch-exporter to 5.1.1

### PR DESCRIPTION
**Pull request by lucianvlad**
**What this PR does / why we need it**:
Upgrade prometheus-elasticsearch-exporter to 5.1.1
**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: fixes #

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:

**Add a screenshot or an example to illustrate the proposed solution:**

**Checklist:**

- [x] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [x] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.
- Chart checklist (pick exactly one):
    - [ ] I upgraded no Chart.
    - [x] I upgraded a Chart and determined that no migration steps are needed.
    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).

**Pipeline config** *(if applicable)*
If you change some config options (e.g. add/rename variable or change the default value) you may need to update the config used by the pipeline in `pipeline/config`.

### CODE REVIEW

**Comment by aarnq** [comment]
Could you add a condition on `kubernetesPSP` in `helmfile/values/prometheus-opensearch-exporter.yaml.gotmpl` for the creation of PSPs? I missed that earlier.

**Comment by lucianvlad** [comment]
> Could you add a condition on `kubernetesPSP` in `helmfile/values/prometheus-opensearch-exporter.yaml.gotmpl` for the creation of PSPs? I missed that earlier.

Added it now
Thank you!

**Review by aarnq — APPROVED** [review]

**Review by davidumea — APPROVED** [review]

File: helmfile/upstream/prometheus-elasticsearch-exporter/values.yaml [hunk 1]
```diff
@@ -144,6 +155,10 @@ es:
   ##
   indices_mappings: true
 
+  ## If true, query stats for aliases.
+  ##
+  aliases: false
```
Comment (crssnd) [inline helmfile/upstream/prometheus-elasticsearch-exporter/values.yaml:160]: is this, `slm` or `data_stream` something that we want to enable?

File: helmfile/upstream/prometheus-elasticsearch-exporter/values.yaml [hunk 2]
```diff
@@ -74,7 +86,6 @@ deployment:
 ##   KEY_2: value2
 env: {}
```
Comment (crssnd) [inline helmfile/upstream/prometheus-elasticsearch-exporter/values.yaml:87]: I think they use a different method to connect to opensearch now, if you check the logs or the grafana dashboard you should see that there are no metrics

**Comment by aarnq** [comment]
Since we've changed it you now need to remove the PSP again :sweat_smile:

**Review by robinAwallace — APPROVED** [review]

**Pull request merged**